esmvaltool/diag_scripts/hydrology/lisflood.py

```diff
@@ -94,11 +97,11 @@ def compute_windspeed(uas, vas):
     return sfc_wind
 
 
-def save(cube, var_name, dataset, cfg):
+def save(xrds, var_name, dataset, cfg):
     """Save processed cube to a lisflood-compatible file."""
-    time_coord = cube.coord('time')
-    start_year = time_coord.cell(0).point.year
-    end_year = time_coord.cell(-1).point.year
+    time_coord = xrds.coords['time']
+    start_year = time_coord[0].dt.year.data
+    end_year = time_coord[-1].dt.year.data
```

Comment:
This is a bit more native xarray:
```suggestion
    start_year = int(xrds.time[0].dt.year)
    end_year = int(xrds.time[-1].dt.year)
```